README.md

```diff
@@ -548,7 +548,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Jessica Salbert](https://github.com/jessicasalbert)
   - [Jessica Teo](https://github.com/jessicateocw)
   - [JesusGerardoAguiar](https://github.com/JesusGerardoAguiar)
-  - [Jivthesh](https://github.com/jivthesh)
+  - [Jivthesh](https://github.com/jivthesh)  
```

Comment:
Please remove this trailing whitespace here.
```suggestion
  - [Jivthesh](https://github.com/jivthesh)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix